renderdoccmd/android/AndroidManifest.xml

```diff
@@ -2,15 +2,16 @@
 <!-- BEGIN_INCLUDE(manifest) -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="@RENDERDOC_ANDROID_PACKAGE_NAME@">
 
-  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="26"/>
+  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="30"/>
```

Comment:
I'm concerned about changing this SDK version. Is this strictly required?

Reply:
We can change the targetSDKVersion back to 26, that should be no issue!